gpflow/posterior.py

```diff
@@ -30,10 +38,19 @@
     SeparateIndependentInducingVariables,
     SharedIndependentInducingVariables,
 )
-from .models.model import MeanAndVariance
+from .types import MeanAndVariance
 from .utilities import Dispatcher
 
 
+class DeltaDist(Module):
```

Comment:
This is an interesting and obscure object :) Could you add a bit more details about what it is for and how to use it?

Reply:
We could remove it again altogether. Happy to hear what you think on this, or whether we could improve on it further - I just wanted to take the opportunity of working on this to see if we can do better than having if:else branches based on whether q_sqrt is None, rank 2, or rank 3..

Reply:
addressed